code/game/machinery/ARES/ARES_procs.dm

```diff
@@ -506,16 +507,18 @@ GLOBAL_LIST_INIT(maintenance_categories, list(
 
 		// -- Emergency Buttons -- //
 		if("general_quarters")
-			if(security_level == SEC_LEVEL_RED || security_level == SEC_LEVEL_DELTA)
-				to_chat(usr, SPAN_WARNING("Alert level is already red or above, General Quarters cannot be called."))
+			if(!COOLDOWN_FINISHED(src, ares_quarters_cooldown))
+				to_chat(usr, SPAN_WARNING("It has not been long enough since last General Quarters call!"))
 				playsound(src, 'sound/machines/buzz-two.ogg', 15, 1)
 				return FALSE
-			set_security_level(2, no_sound = TRUE, announce = FALSE)
+			if(security_level != SEC_LEVEL_RED && security_level != SEC_LEVEL_DELTA)
+				set_security_level(2, no_sound = TRUE, announce = FALSE)
```

Comment:
use defines for this proc. it should be 
set_security_level(SEC_LEVEL_RED, no_sound = TRUE, announce = FALSE)